.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+ENV VLLM_QUANTIZE_Q40_LIB="/usr/local/lib/python3.12/dist-packages/vllm_int4_for_multi_arc.so"
+ENV VLLM_OFFLOAD_WEIGHTS_BEFORE_QUANT=1
+ENV VLLM_ALLOW_LONG_MAX_MODEL_LEN=1
+ENV UR_L0_USE_IMMEDIATE_COMMANDLISTS=0
+
+RUN pip uninstall oneccl oneccl-devel -y
+
+# Cleanup tmp files
+RUN rm -rf /tmp/*
+
+# Configure environment to source oneAPI
+RUN echo "source /opt/intel/oneapi/setvars.sh --force" >> /root/.bashrc
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)